packages/passwordless-auth-construct/src/custom-auth/custom_auth_service.ts

```diff
@@ -397,4 +468,41 @@ export class CustomAuthService {
       (entry) => !entry.challengeMetadata?.includes('PROVIDE_AUTH_PARAMETERS')
     ).length;
   }
+
+  /**
+   * Update user on Cognito UserPool by mark attribute as verified
+   * @param username - The username to be verify the attribute
+   * @param attributeName - The attribute that is going to be mark as verified
+   * @param region - The UserPool region
+   * @param userPoolId - The UserPool ID
+   */
+  private async markAsVerifiedAndDeletePasswordlessAttribute(
```

Comment:
request: 
1. move this function to `create_user_service.ts`
2.`CognitoIdentityProviderClient` can be a property passed to the class constructor. This should allow dependency injection and resuse for the create user logic.